wormhole-connect/src/views/Bridge/Inputs/TokenWarnings.tsx

```diff
@@ -205,10 +215,24 @@ function TokenWarnings() {
   ]);
 
   useEffect(() => {
+    // check if the token is USDC and the chains involved are not CCTP
+    const usdcAndNoCCTP =
```

Comment:
also, we may want this to be configurable to allow integrators to enable/disable at their convenience, not sure if all wants to get redirect to portal bridge

Reply:
Done!!